addons/godot_pixelorama_importer/pixelorama_file_open/open.gd

```diff
@@ -0,0 +1,70 @@
+extends EditorInspectorPlugin
+
+var tex:StreamTexture
+var user_config_req:bool = true
+var pixel_path:String
+
+var ctl
+
+func _init():
+	#load config file settings
+	var cfg = ConfigFile.new()
+	var err = cfg.load("user://pixel_plugin.ini")
+	if err == 7:
+		cfg.save("user://pixel_plugin.ini")
+		err = cfg.load("user://pixel_plugin.ini")
+	if err == OK:
+		if cfg.has_section_key("location","pixelorama"):
+			pixel_path = cfg.get_value("location","pixelorama")
+			user_config_req = false
+	else:
+		print("Error loading Plugin \n", err)
+
+func can_handle(object):
+	if object is StreamTexture:
+		tex = object
+		return true
+	return false
+
+func parse_end():
+	ctl = preload("inspector_menu.tscn").instance()
+	var btn = ctl.get_node("VBoxContainer/OpenButton")
+	btn.connect("pressed",self,"open_btn_pressed")
+	btn.disabled = user_config_req
+	var win = ctl.get_node("WindowDialog")
+	win.connect("confirmed",self,"_folder_selected")
+	ctl.get_node("VBoxContainer/ConfigButton").connect("pressed",self,"_on_ConfigButton_pressed", [win])
+	add_custom_control(ctl)
+
+func open_btn_pressed():
+	var tex_path = ProjectSettings.globalize_path(tex.resource_path)
+	var extension = ""
+	if OS.get_name() == "X11":
+		extension = ".x86_64"
+	elif OS.get_name() == "Windows":
+		extension = ".exe"
+	else: 
+		push_warning("PixeloramaPlugin: Unsupported Platform")
+		return
+	var f:File = File.new()
+	if !f.file_exists(pixel_path+"/Pixelorama"+ extension):
```

Comment:
Concatenate "/Pixelorama" to the pixel_path once so it's not repeated multiple times :P

Reply:
I am using `pixel_path` in other places so cant override. Although making a separate variable makes sense, its only used twice. So it isn't needed.